modules/monitoring-configuring-remote-write-storage.adoc

```diff
@@ -195,7 +186,25 @@ data:
         - sourceLabels: [__name__]
           regex: 'my_metric'
           action: keep
-
+----
++
+.Example of forwarding metrics called `my_metric_1` and `my_metric_2` in `my_namespace` namespace
```

Comment:
I added the filter by namespace example only for UWM as I was not sure if it made sence for core platform monitoring. Please let me know. Thank you! :pray:

Reply:
hmm the example below applies to platform monitoring?

Reply:
Oh I just realised I forgot to change namespace and config map name, my bad :pray:  I will fix that right away

Reply:
@simonpasquier It is not clear to me if the namespace filter makes sence for core platform monitoring, wouldn't it automatically send metrics from openshift-monitoring namespace?

Reply:
no it makes sense for cluster monitoring too.